tests/login.test.js

```diff
@@ -0,0 +1,40 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import loginPage from '../pages/login.page';
+import homePage from '../pages/home.page';
+import { expect } from 'chai';
+
+describe('OpenBank', () => {
+
+    
+    it('Should access the home page when attempting to login with valid credentials', async () => {
+        await loginPage.open('login');
+        let username = 'jsmith@demo.io';
+        let password = 'Demo123!';
+        await loginPage.enter(username, password);
+       
+        await assert.equal(await $('h1').getText(),'Dashboard','Error: it did not show the expected text');
+        //expect(await homePage.getDashboardTitle()).to.equal('Dashboard');
+        
+     });
+     it('Should show an error when attempting to login with invalid credentials', async () => {
+        //await loginPage.open('login');
```

Comment:
Same comment as above. This looks like a proper implementation of navigating to login page. Why is it commented? It's also preferable over direct `browser.url('login')`, as it's working at pageObject's abstraction level.

Reply:
fixed...